draft-ietf-tls-dtls-connection-id.md

```diff
@@ -366,7 +366,10 @@ in changing a peer address if they are able to rewrite source addresses
 and if replayed packets are able to arrive before any original. 
 
 - There is a strategy for ensuring that the new peer address is able to 
-receive and process DTLS records. No such test is defined in this	specification. 
+receive and process DTLS records. No such test is defined in this specification.
+
+If any of the above checks fails, the receiver MUST silently discard the
```

Comment:
This doesn't mention the main fear, that a message is sent to the previous not updated address.

The "any of the above checks" may cause irritation, though the check for the newer address does not inhibit the proper processing of a "out of order" record on application layer.

Reply:
I am also not sure what this sentence means. Whcih checks?

Reply:
> I am also not sure what this sentence means. Whcih checks?

the three conditions in L357-372

Reply:
I have updated the text to clarify that we are talking about the conditions above.